<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QDialog" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="friendsList">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>        Add a friend</string>
   </property>
   <widget class="QPushButton" name="Friendbutton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add friend</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AddFriendName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Friends list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>281</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>                                         Chat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You can select a friend to chat with!&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteAccountButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete 
account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JoinLectureChatButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Join lecture
chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DownloadButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download
lecture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LectureAddButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add lecture</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteLectureButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete
lecture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MakeAdminButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveAdminButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove
admin</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>QuitButton</zorder>
  <zorder>friendsList</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>SendButton</zorder>
  <zorder>pushButton</zorder>
  <zorder>DeleteAccountButton</zorder>
  <zorder>JoinLectureChatButton</zorder>
  <zorder>DownloadButton</zorder>
  <zorder>LectureAddButton</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>DeleteLectureButton</zorder>
  <zorder>MakeAdminButton</zorder>
  <zorder>RemoveAdminButton</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>QuitButton</tabstop>
  <tabstop>AddFriendName</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>textBrowser_2</tabstop>
  <tabstop>friendsList</tabstop>
  <tabstop>SendButton</tabstop>
  <tabstop>Friendbutton</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
